plugins/module_utils/rundeck.py

```diff
@@ -82,6 +81,6 @@ def api_request(module, endpoint, data=None, method="GET", content_type="applica
             json_response = json.loads(content)
             return json_response, info
     except AttributeError as error:
-        module.fail_json(msg="Rundeck API request error", exception=to_native(error), execution_info=info)
+        module.fail_json(msg=f"Rundeck API request error: {error}", exception=error, execution_info=info)
```

Comment:
This is not allowed and will crash the module in ansible-core 2.17 and 2.18. Passing exception objects is ONLY allowed from 2.19 on.